debian/changelog

```diff
@@ -0,0 +1,5 @@
+golang-logrus (0.6.0) UNRELEASED; urgency=medium
+
+	* Initial release.
```

Comment:
Haaawt -- now if you file that ITP bug (with `X-Debbugs-CC: paultag@debian.org, admwiggin@gmail.com`), this'll just change to `* Initial release (Closes: #xxxx).` where `xxxx` is the number of the ITP bug! :smile: